### PR TITLE
Fix azure artifacts path download

### PR DESCRIPTION
**Pull request by victorcmsantos**
Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Comment by victorcmsantos** [comment]
it is ok

File: pulpcore/content/handler.py [hunk 1]
```diff
@@ -601,7 +601,7 @@ def _serve_content_artifact(self, content_artifact, headers):
             artifact_file = content_artifact.artifact.file
             content_disposition = f"attachment;filename={content_artifact.relative_path}"
             parameters = {"ResponseContentDisposition": content_disposition}
-            url = artifact_file.storage.url(artifact_file.name, parameters=parameters)
+            url = artifact_file.storage.url(artifact_file.name)
```
Comment (daviddavis) [inline pulpcore/content/handler.py:604]: We need to support `ResponseContentDisposition` for s3. I think we need something like this:

```python
        if settings.DEFAULT_FILE_STORAGE == "pulpcore.app.models.storage.FileSystem":
            filename = content_artifact.artifact.file.name
            return FileResponse(os.path.join(settings.MEDIA_ROOT, filename), headers=headers)
        elif settings.DEFAULT_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage"
            artifact_file = content_artifact.artifact.file
            content_disposition = f"attachment;filename={content_artifact.relative_path}"
            parameters = {"ResponseContentDisposition": content_disposition}
            url = artifact_file.storage.url(artifact_file.name, parameters=parameters)
            raise HTTPFound(url)
        elif settings.DEFAULT_FILE_STORAGE == "storages.backends.azure_storage.AzureStorage"
            artifact_file = content_artifact.artifact.file
            url = artifact_file.storage.url(artifact_file.name)
            raise HTTPFound(url)
        else:
            raise NotImplementedError()
```

**Comment by daviddavis** [comment]
Can you file an issue here: https://pulp.plan.io/projects/pulp/issues/new

And then attach it to your commit and add a changelog? More info: https://docs.pulpproject.org/pulpcore/contributing/pull-request-walkthrough.html.

File: pulpcore/content/handler.py (hunk 1 above)
Comment (victorcmsantos) [reply pulpcore/content/handler.py:604]: Should I create a new pull request with this code ?

Comment (daviddavis) [reply pulpcore/content/handler.py:604]: I would just use this PR. Update the code then run:

```
git add -A .
git commit --amend
git push -f
```

**Comment by dralley** [comment]
Was reopened here https://github.com/pulp/pulpcore/pull/1137

**Pull request closed without merging**